### PR TITLE
Fix facebook login

### PR DESCRIPTION
**Pull request by atalyaalon**

### CODE REVIEW

File: anyway/config.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 # variables
 #
 
+SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL')
+APP_SECRET_KEY = os.environ.get('APP_SECRET_KEY')
```
Comment (elazarg) [inline anyway/config.py:12]: How about using the `parsers.secrets` module?

Comment (atalyaalon) [reply anyway/config.py:11]: @elazarg do you mean [this module](https://docs.python.org/3/library/secrets.html)?

Comment (elazarg) [reply anyway/config.py:11]: I mean [this module](https://github.com/hasadna/anyway/blob/dev/anyway/parsers/secrets.py).

Comment (atalyaalon) [reply anyway/config.py:11]: Yes, but I suggest we'll do it in another pr

Comment (atalyaalon) [reply anyway/config.py]: @elazarg actually I added it - did I use it correctly?
https://github.com/hasadna/anyway/pull/1327/commits/f22978fc9c7877ae72bb5bcd672c0f508fb6f119

Comment (elazarg) [reply anyway/config.py]: Yes, though of course if you can test that it'd be better.

**Pull request merged**